manifests/init.pp

```diff
@@ -1,3 +1,29 @@
+# Install and manage the NATS messaging system
+#
+# @param routes_password
+# @param servers
+# @param client_port
+# @param monitor_port
+# @param cluster_port
+# @param max_payload_size
+# @param max_pending_size
+# @param max_connections
+# @param debug
+# @param trace
+# @param announce_cluster
+# @param binpath
+# @param configdir
+# @param piddir
+# @param binary_source
+# @param service_name
+# @param service_ensure
+# @param service_type
+# @param cert_file
+# @param key_file
+# @param ca_file
+# @param manage_collectd
+# @param limit_nofile
+#
```

Comment:
This is a struggle I've been having myself. I'm not spun up on NATS enough to actually write the docs, so I'm going to agree with you.